runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -234,15 +234,20 @@ public void onItemSpawned(ItemSpawned itemSpawned)
 			// The spawn time remains set at the oldest spawn
 		}
 
-		boolean shouldNotify = !config.onlyShowLoot() && config.highlightedColor().equals(getHighlighted(
+		final Color highlightedColor = getHighlighted(
 			groundItem.getName(),
 			groundItem.getGePrice(),
-			groundItem.getHaPrice()));
+			groundItem.getHaPrice());
+		boolean shouldNotify = !config.onlyShowLoot() && config.highlightedColor().equals(highlightedColor);
 
 		if (config.notifyHighlightedDrops() && shouldNotify)
 		{
 			notifyHighlightedItem(groundItem);
 		}
+		else if (highlightedColor != null && (!config.onlyShowLoot() || groundItem.isMine()) && getGroundItemConfigValue(groundItem) > config.getNotifyOverValue())
```

Comment:
Why does this require `highlightedColor != null`?

Reply:
I added this to ensure that items that are explicitly hidden would not trigger valuable notifications, but now I see this would prevent notifications from working if their price checkpoints are lower than their notification threshold.

Should I still prevent items that are explicitly hidden from triggering this notification?